.github/workflows/get-secretmanager-secrets.yml

```diff
@@ -0,0 +1,21 @@
+name: get-secretmanager-secrets Unit
+on: [push, pull_request]
+jobs:
+  run:
+    name: test
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [ubuntu-latest, windows-latest, macos-latest]
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-node@master
+      with:
+        node-version: 12.x
+
+    - run: npm install
+      working-directory: ./get-secretmanager-secrets
+
+    - run: npm test
+      working-directory: ./get-secretmanager-secrets
```

Comment:
Also suggest including a lint check similar to the setup-gcloud action: https://github.com/GoogleCloudPlatform/github-actions/blob/9ef0382a75df2efcc72850f4e963927a45d59414/.github/workflows/setup-gcloud.yml#L23